tests/contrib.bats

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "bud with a path to a Dockerfile (-f) containing a non-directory entry" {
+  run_buildah bud -t buildah ../contrib/buildahimage/stable
```

Comment:
I don't see this working in CI, because the `contrib` directory isn't shipped in the `-tests` rpm

Reply:
Can I just turn it off for gating tests?